### PR TITLE
Enable support for adjusting SSH server

### PR DESCRIPTION
**Pull request by igorpecovnik**
# Description

Port SSHD server configuration from armbian-config. I hope this is all that needs to be done?

# Testing Procedure

- [x] Configuring key only login
- [x] Configuring 2FA (here is a small problem. I didn't find a way how to have: key + OTP. It only works password + OTP
  
# Checklist

- [x] Changes have been tested and verified
- [x] I have included necessary metadata in the code, including associative arrays

### CODE REVIEW

**Comment by igorpecovnik** [comment]
Probably better to move this to section system?

**Comment by Tearran** [comment]
No issue having in system, but

sub menu is quite buggy 
Produces Errors 
breaks --doc :
does not display in --help 
not available for --cli

**Comment by igorpecovnik** [comment]
> Produces Errors
breaks --doc :
does not display in --help
not available for --cli

OK. will look into.

File: lib/armbian-configng/config.ng.jobs.json [hunk 1]
```diff
@@ -75,6 +75,145 @@
...
+                            "condition": "grep -q 'PubkeyAuthentication no' /etc/ssh/sshd_config"
+                            },
+                            {
+                            "id": "S14",
+                            "description": "Disable OTP authentication",
+                            "command": [
+                                           "clear",
+                                           "! check_if_installed libpam-google-authenticator && ! check_if_installed qrencode || debconf-apt-progress -- apt-get -y purge libpam-google-authenticator qrencode",
+                                           "sed -i \"s/^#\\?ChallengeResponseAuthentication.*/ChallengeResponseAuthentication no/\" /etc/ssh/sshd_config || sed -i \"0,/KbdInteractiveAuthentication/s//ChallengeResponseAuthentication yes/\" /etc/ssh/sshd_config" ,
+                                           "sed -i '/^auth required pam_google_authenticator.so nullok/ d' /etc/pam.d/sshd",
```
Comment (Tearran) [inline lib/armbian-configng/config.ng.jobs.json:168]: Putting commands here will work, But will not be available though the --api feature 

example : try
```
sudo ./configng/bin/armbian-configng --api see_ping
``` 

```
sudo ./configng/bin/armbian-configng --api 
 check_ip_version
```

**Comment by Tearran** [comment]
Cleaned up a little, fixed the error and included this in #51

**Review by Tearran — APPROVED** [review]
good addition

**Pull request merged**